src/components/widgets-common/chart-threshold/chart-threshold.component.coffee

```diff
@@ -123,7 +123,9 @@ module.component('chartThreshold', {
       return unless validateHistParameters()
       Highcharts.addEvent(chart.hc.container, 'click', onChartClick)
       _.each buildThresholdsFromKpis(), (threshold)->
-        ctrl.chart.addThreshold(threshold)
+        thresholdSerie = ctrl.chart.findThreshold(threshold.kpiId)
```

Comment:
I wouldn't use the kpi id as the key here: couldn't we in the future add thresholds to a chart without them be related to a kpi?
Maybe use the series name instead?